qa/tasks/cephfs/test_volume_client.py

```diff
@@ -18,7 +18,8 @@ class TestVolumeClient(CephFSTestCase):
 
     def setUp(self):
         CephFSTestCase.setUp(self)
-        self.py_version = self.ctx.config.get('overrides', {}).get('python', 'python')
+        self.py_version = self.ctx.config.get('overrides', {}).\
+                          get('python', self.py_version)
```

Comment:
We may have lost a related change but I don't see a difference in behavior for the code?

Reply:
I don't want us to hardcode default to 'python'. Default is declared [here](https://github.com/ceph/ceph/blob/master/qa/tasks/cephfs/test_volume_client.py#L17) and, IMO, it should be declared only once (whether it is 'python', 'python2' or 'python3'). Therefore, I am replacing 'python' by self.py_version.

Reply:
IIRC, I hit this issue when I wanted to use python3 locally for running `test_volume_client.py`

Reply:
My point is that class variable self.py_version is defined as `'python'` so there should be no difference here.

Reply:
`self.py_version` will be changed to `python3` in PR #27718.

Reply:
So you want the default to be python3. I don't like how we're shadowing the class variable here. Can we be explicit: `... get('python', TestVolumeClient.py_version)` and maybe even rename the class variable to `default_py_version`.

Reply:
Cool, didn't want to shadow class variable but I failed to notice it. I've incorporated both the changes.